packagemanifests/integreatly-operator/2.10.0/integreatly-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,622 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
@matskiv is it necessary to bump integreatly-operator to 2.10 here. I think this would normally be done at release time?

Reply:
Yes, you are absolutely right.
I committed this change by mistake, and it's reverted now.